components/common/UserPicture.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { Component } from 'react';
```

Comment:
Isn't it save to remove "React" as an import, as it's not used anymore in the entire file?